src/utils/utils_cleaner.c

```diff
@@ -51,7 +51,7 @@ static struct ocf_request *_ocf_cleaner_alloc_req(struct ocf_cache *cache,
 
 	/* Allocate pages for cleaning IO */
 	req->data = ctx_data_alloc(cache->owner,
-			ocf_line_size(cache) / PAGE_SIZE * count);
+			count * ocf_line_size(cache) / PAGE_SIZE);
```

Comment:
you would need to divide with rounding UP to make sure there is enough space in the data in case of (count * ocf_line_size) not being multiplicity of PAGE_SIZE

Reply:
@arutk , will this be ok with `round_up(ocf_line_size(cache), PAGE_SIZE) / PAGE_SIZE * count` ?

Reply:
yes, I think the macro name in ocf is OCF_DIV_ROUND_UP

Reply:
OCF_DIV_ROUND_UP((uint64_t)count * ocf_line_size(cache), PAGE_SIZE);

Reply:
... casting to 64bit just to be safe from any overflows

Reply:
thanks, force pushed with OCF macro.